### ISSUE TITLE
Power poles and layers

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/iD#8100]
If I turn on "Power Features" I can see all the power features, including power poles and towers, but to add one, I have to turn on "POI" layers, it's strange.

**Comment by quincylvania** [comment]
@maro-21 Yeah, I think iD might group all standalone points into the POI layer even if they'd make more sense somewhere else…

**Comment by maro-21** [comment]
Another example with `landuse=construction`. What I meant is that if I turn on the "future/past" layer, I can see and edit construction sites, but to add one, I have to turn on the "landuse" layer. The same here with power poles: I can edit the existing ones if I turn on "power feature", but to add one, I have to turn on "POI" layers.

Is this behavior ok?

**Comment by matkoniecz** [comment]
it looks like https://github.com/openstreetmap/iD/blob/f154bebafd3e3638942ea0048a29a0162227d5bb/modules/renderer/features.js#L106 would need to be modified
